README.md

```diff
@@ -241,9 +289,9 @@ sudo setcap cap_net_bind_service=+ep leapchat
 
 ## Documentation Links
 
-Currently, some of the packages we use are slightly out of date. Here are some quick documentation links that might prove useful:
+[Bootstrap 5](https://getbootstrap.com/docs/5.0/getting-started/introduction/)
 
-[React Bootstrap version 0.33.1 docs](https://react-bootstrap-v3.netlify.app/components/tooltips/)
+[React Bootstrap 2](https://react-bootstrap.github.io/)
```

Comment:
These links are the only update we really need to go along with this branch. The rest I've just been meaning to do for a while.